README.md

```diff
@@ -14,7 +14,7 @@ In the next section, I define atomic comparison and resulting patterns, then des
 ### Outline of my approach
 
 
-Proposed algorithm is a comparison-first alternative to deep learning, neither statistical nor neuromorphic. It's a search for patterns of incremental composition in a recursively extended pipeline. Each level of this pipeline cross-compares inputs and then clusters them into patterns by proximity and resulting match. So, pattern is a set of nearest-neighbor matching inputs. In more common use, pattern is a matching set of inputs. But that's really a second-order covariance pattern: a set of inputs with matching derivatives from prior comparisons.
+The proposed algorithm is a comparison-first alternative to deep learning, neither statistical nor neuromorphic. It's a search for patterns of incremental composition in a recursively extended pipeline. Each level of this pipeline cross-compares inputs and then clusters them into patterns by proximity and resulting match. So, the pattern is a set of nearest-neighbor matching inputs. In more common useage, a pattern is a matching set of inputs. But that's really a second-order covariance pattern: a set of inputs with matching derivatives from prior comparisons.
```

Comment:
Ok, "usage" is better, thanks. But those articles mostly add clutter. We are talking about definitions here, so it can be "a" or "the" depending on POV.